Cargo.toml

```diff
@@ -14,8 +14,8 @@ readme = "README.md"
 
 [dependencies]
 console_log = "1.0.0"
-leptos = { version = "0.5.4", features = ["csr", "nightly"] }
-leptos-use = { version = "0.9.0", features = ["serde"] }
+leptos = { version = "0.6.5", features = ["csr", "nightly"] }
```

Comment:
Amazing. What are your thoughts on leaving `nightly`? 

When we fix #1, we can actually remove the `csr` feature :)

Reply:
I think nightly is fine for now. It builds for now at least...